Demo/Demo/SwiftUIComponents/CardPaymentViews/CardOrderApproveView.swift

```diff
@@ -0,0 +1,86 @@
+import SwiftUI
+import CardPayments
+import CorePayments
+
+struct CardOrderApproveView: View {
+
+    @State private var cardNumberText: String = "4111 1111 1111 1111"
+    @State private var expirationDateText: String = "01 / 25"
+    @State private var cvvText: String = "123"
+    let orderID: String
+    @ObservedObject var cardPaymentViewModel: CardPaymentViewModel
```

Comment:
Do we have a style guide for the order of State/ObservableObjects/properties? It doesn't look like Google or Airbnb define how property wrappers are handled when it comes to file order. Curious if you or @scannillo have thoughts on how we should order these.

Reply:
Oh I think I've seen @ObservedObject on top and then any property wrappers and then instance properties order most often but I don't know if there is any agreed upon convention.